settings/Case.setting.php

```diff
@@ -100,4 +100,19 @@
     'description' => 'How to sort activity-types on the "Manage Case" screen? (Set "Default" to load setting from the legacy "Settings.xml" file.)',
     'help_text' => '',
   ),
+  'civicaseActivityRevisions' => array(
+    'group_name' => 'CiviCRM Preferences',
+    'group' => 'core',
+    'name' => 'civicaseActivityRevisions',
+    'type' => 'Boolean',
+    'quick_form_type' => 'YesNo',
+    'default' => TRUE,
```

Comment:
Hmm, I was thinking mostly about new installs, I think it should be TRUE for them. FALSE is ok for existing installs, agree that's safer.